config/src/config/network_config.rs

```diff
@@ -74,7 +104,12 @@ impl Default for NetworkConfig {
 impl NetworkConfig {
     pub fn network_with_id(network_id: NetworkId) -> NetworkConfig {
         let mut config = Self {
-            connectivity_check_interval_ms: 5000,
+            max_connection_delay_ms: MAX_CONNECTION_DELAY_MS,
```

Comment:
btw I feel the mixing of consts and na default config seem kinda duplicitous but ... that's my style

Reply:
Yeah, I'm much more a readability thing to see the constants.  I know it's kinda a lot of repetitiveness.